pkg/resources/schema.go

```diff
@@ -264,11 +264,13 @@ func SchemaExists(data *schema.ResourceData, meta interface{}) (bool, error) {
 		return false, err
 	}
 
+	exists := false
```

Comment:
Seems like it would be simpler to just use `defer rows.Close()`?

Reply:
Okay, I'll update tomorrow.

Reply:
actually I did a pass to add this everywhere that I think is relevant- https://github.com/chanzuckerberg/terraform-provider-snowflake/pull/74